recipes/geventhttpclient/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "geventhttpclient" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: bd87af8854f5fb05738916c8973671f7035568aec69b7c842887d6faf9c0a01d
+  - url: https://raw.githubusercontent.com/gwik/{{ name }}/master/LICENSE-MIT
```

Comment:
You could also just include this in the recipe dir for simplicity in future.

Reply:
It also appears the next release will [include its LICENSE](https://github.com/gwik/geventhttpclient/commit/04f9f606d1623823a6ce639043c1af0fd98454a2) in the source distribution.